api/pkg/store/mongo/migrations.go

```diff
@@ -9,6 +9,11 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+func renameCollumn(db *mongo.Database, collectionName, collumnOrig, collumnDest string) error {
```

Comment:
`renameCollumn` refers to a table-based database.
`renameField` seems more appropriate for a document-based database.

Reply:
Good catch! I made this change locally and will see if I can improve it more before resending it.